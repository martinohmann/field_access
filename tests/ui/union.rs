use field_access::FieldAccess;

#[derive(FieldAccess)]
pub union Union {
    a: i64,
    b: u8,
}

fn main() {}
