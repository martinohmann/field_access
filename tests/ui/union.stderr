error: FieldAccess does not support unions
 --> tests/ui/union.rs:4:1
  |
4 | / pub union Union {
5 | |     a: i64,
6 | |     b: u8,
7 | | }
  | |_^
